api_app/models/domain/resource.py

```diff
@@ -16,14 +16,28 @@ class ResourceType(str, Enum):
     SharedService = strings.RESOURCE_TYPE_SHARED_SERVICE
 
 
+class NewResourceHistoryItem(AzureTREModel):
+    """
+    Resource History Item - to preserve history of resource properties
+    """
+    id: str = Field(title="Id", description="GUID identifying the resource history item")
+    resourceId: str = Field(title="Id", description="GUID identifying the resource id")
+    properties: dict = Field({}, title="Resource template parameters", description="Parameters for the deployment")
+    isEnabled: bool
+    resourceVersion: int
+    updatedWhen: float = 0
+    user: dict = {}
+    templateVersion: Optional[str]
+
+
 class ResourceHistoryItem(AzureTREModel):
     """
     Resource History Item - to preserve history of resource properties
     """
-    properties: dict = {}
+    properties: dict = Field({}, title="Resource template parameters", description="Parameters for the deployment")
     isEnabled: bool
     resourceVersion: int
-    updatedWhen: float
+    updatedWhen: float = 0
```

Comment:
why is this needed

Reply:
To match it with default values like the rest of the models